### DISCUSSION TITLE
Translations for main menu and margin attributes

### DISCUSSION THREAD

**Discussion by amerr-k** [GrapesJS/grapesjs#4185] [Q&A]
Is it possible to translate main menu in locale folder? I'm using translations for bosnian language (it's the same for other languages) and main menu doesn't have translations for its actions. Only thing that works in my case is that I can modify index.js from its plugin "grapesjs-preset-newsletter" and translations will be there but I don't want that.
![image](https://user-images.githubusercontent.com/25527072/157218755-43f86abf-5ec9-48b0-a866-5cfcfca058f0.png)
Also can't find a way to translate positions of margins and paddings even though they have translations in locale.
![image](https://user-images.githubusercontent.com/25527072/157219359-7e54eee3-095d-4770-8956-0777517ded46.png)

**Comment by artf** [comment]
Here you can find the documentation about i18n: https://grapesjs.com/docs/modules/I18n.html
and here the EN file as a reference: https://github.com/artf/grapesjs/blob/master/src/i18n/locale/en.js
